Source/Core/Matrix4.js

```diff
@@ -1675,7 +1675,7 @@ const scaleScratch5 = new Cartesian3();
  * Extracts the rotation matrix assuming the matrix is an affine transformation.
  *
  * @param {Matrix4} matrix The matrix.
- * @param {Matrix4} result The object onto which to store the result.
+ * @param {Matrix3} result The object onto which to store the result.
  * @returns {Matrix4} The modified result parameter.
```

Comment:
Sorry that this comment is late, but it looks like the `@returns` should also be a `Matrix3` as well?

Reply:
Tweaked in https://github.com/CesiumGS/cesium/commit/77d458504b8e4d22cf860a21610b144b5e4b0e6b.